2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_setup.py:_flush():76] Configure stats pid to 2944
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_setup.py:_flush():76] Loading settings from C:\Users\debryu\.config\wandb\settings
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_setup.py:_flush():76] Loading settings from C:\Users\debryu\Desktop\VS_CODE\HOME\Deep_Learning\visual_grounding\visual-grounding\Project\wandb\settings
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Project\\distil_model.py', 'program': 'distil_model.py'}
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_init.py:_log_setup():507] Logging user logs to C:\Users\debryu\Desktop\VS_CODE\HOME\Deep_Learning\visual_grounding\visual-grounding\Project\wandb\run-20230505_201431-qf10cam2\logs\debug.log
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_init.py:_log_setup():508] Logging internal logs to C:\Users\debryu\Desktop\VS_CODE\HOME\Deep_Learning\visual_grounding\visual-grounding\Project\wandb\run-20230505_201431-qf10cam2\logs\debug-internal.log
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_init.py:init():547] calling init triggers
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_init.py:init():595] starting backend
2023-05-05 20:14:31,440 INFO    MainThread:2944 [wandb_init.py:init():599] setting up manager
2023-05-05 20:14:31,456 INFO    MainThread:2944 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-05-05 20:14:31,456 INFO    MainThread:2944 [wandb_init.py:init():605] backend started and connected
2023-05-05 20:14:31,471 INFO    MainThread:2944 [wandb_init.py:init():695] updated telemetry
2023-05-05 20:14:31,549 INFO    MainThread:2944 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-05-05 20:14:31,925 ERROR   MainThread:2944 [wandb_init.py:init():758] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2023-05-05 20:14:32,018 ERROR   MainThread:2944 [wandb_init.py:init():1163] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "C:\Users\debryu\miniconda3\envs\pytorch\lib\site-packages\wandb\sdk\wandb_init.py", line 1145, in init
    run = wi.init()
  File "C:\Users\debryu\miniconda3\envs\pytorch\lib\site-packages\wandb\sdk\wandb_init.py", line 764, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2023-05-05 20:14:33,018 WARNING MsgRouterThr:2944 [router.py:message_loop():77] message_loop has been closed
